webpack.config.js

```diff
@@ -65,6 +67,7 @@ module.exports = {
   devtool: 'cheap-source-map',
   plugins: [
     new DefinePlugin({
+      'process.env.NODE_ENV': JSON.stringify(buildEnv),
```

Comment:
Allows Vue.js build process to strip out development-mode code. See [the Vue.js deployment guide](https://vuejs.org/v2/guide/deployment.html#With-Build-Tools).